Godeps/Godeps.json

```diff
@@ -305,45 +343,45 @@
 		},
 		{
 			"ImportPath": "github.com/opencontainers/image-spec/identity",
-			"Comment": "v1.0.0-rc6",
-			"Rev": "1a6593ab6c3ab5902072b4694a22ff19425396ae"
+			"Comment": "v1.0.0-rc6-12-g372ad78",
+			"Rev": "372ad780f63454fbbbbcc7cf80e5b90245c13e13"
```

Comment:
Just a note here.. containerd is not on the GA version of the image spec but they are on the GA version of the runtime spec and there were pretty major changes to the OCI spec in the months that it went GA.